metrics/metrics.go

```diff
@@ -111,14 +108,34 @@ func parseMetricsFile(path string) (*File, error) {
 	metrics := make(map[string]string)
 
 	for k, v := range tmpMetrics {
-		s, err := json.Marshal(v)
-		if err != nil {
-			l.Errorw("error during marshalling metric value to JSON, skipping",
-				zap.Any("value", v),
-				zap.Error(err))
-			return nil, err
+		switch reflect.TypeOf(v).Kind() { //nolint:exhaustive
+		case reflect.String:
+			if vs, ok := v.(string); ok {
+				// handle special case when "true/false" are written as string
+				if strings.ToLower(vs) == "true" {
```

Comment:
Not sure, but probably it makes sense to consider `strconv.ParseBool` here

Reply:
done